eq-author/cypress/integration/authenticated/validation_spec.js

```diff
@@ -162,6 +172,7 @@ describe("Answer Validation", () => {
           toggleCheckboxOff("@maxValueToggle");
           toggleCheckboxOn("@maxValueToggle");
           cy.get(numberInputId).should("have.value", "3");
+          closeValidationModal();
```

Comment:
Can this be moved to the `afterEach` on line 169?

Reply:
Unfortunately, the `...set previous answer` test does not require `closeValidationModal()`.